### DISCUSSION TITLE
Taro示例项目：vue3+vuex，运行报错

### DISCUSSION THREAD

**Discussion by zhuyifang** [NervJS/taro#9668] [Q & A] [answered]
环境：mac

使用的是官方提供的vue3+vuex模板

运行：npm run build:rn -- --watch

访问：http://127.0.0.1:8081/index.bundle?platform=ios&dev=true

报错如下：

` BUNDLE  ./index 

error: node_modules/metro/src/node-haste/DependencyGraph/assets/empty-module.js: [BABEL] /Users/yfzhu/PROJECT/suanming-taro/node_modules/metro/src/node-haste/DependencyGraph/assets/empty-module.js: Value "vue3" of option "framework" is not supported for React-Native (While processing: "/Users/yfzhu/PROJECT/suanming-taro/node_modules/babel-preset-taro/index.js")
`

**Comment by Chen-jj** [comment, marked as answer]
Vue 不支持编译到 RN

**Comment by Mrwyy** [comment]
我在Taro里引用Vuex 直接报错了 请问你是怎么引用的
